components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -20,14 +22,16 @@ describe('LeagueEntries', () => {
   });
 
   it('renders as if the user made a pick', () => {
-    render(<LeagueEntries entryName="Entry 2" linkUrl='' isPickSet={true} />);
+    render(<LeagueEntries entryName="Entry 2" linkUrl="" isPickSet={true} />);
 
     const leagueEntryContainerCard = screen.getByTestId(
       'league-entry-container-card',
     );
     const leagueEntryNumber = screen.getByTestId('league-entry-number');
     const entryStatus = screen.getByTestId('entry-status');
-    const leagueEntryPickButton = screen.getByTestId('league-entry-pick-button');
+    const leagueEntryPickButton = screen.getByTestId(
+      'league-entry-pick-button',
```

Comment:
Same as above.

Reply:
When saving the code, VS auto populates the comma.